scrapy/__init__.py

```diff
@@ -22,7 +22,7 @@
 
 
 # Scrapy and Twisted versions
-__version__ = pkgutil.get_data(__package__, 'VERSION').decode('ascii').strip()
+__version__ = (pkgutil.get_data(__package__, "VERSION") or b"").decode("ascii").strip()
```

Comment:
:lipstick: Why the change of quotes? I see the line below also uses single quotes.

Reply:
It's a subtle step towards #4658 :wink: